### DISCUSSION TITLE
如何修改electron-builder配置，将需要的文件打包到安装包？/  How to modify the electron-builder configuration and package the required files into the installation package?

### DISCUSSION THREAD

**Discussion by soandsoprogrammer** [electron-vite/electron-vite-vue#350] [Q&A] [answered]
我想配置extraResources这个属性打包一个文件供安装后的程序使用， 我该如何修改electron-builder配置？

I want to configure the property extraResources to package a file for use by the installed program. How can I modify the configuration of electron-builder?

**Comment by soandsoprogrammer** [comment, marked as answer]
哦， 我傻了， 直接改json就行

**Reply by Hevienz** [reply]
能分享一下如何操作吗？

**Reply by FominArtmind** [reply]
I've encountered the same problem and can explain the solution.
By "renaming to json" I think they mean "electron-builder.json5" file. You you rename it to "electron-builder.json", Visual Studio Code will start to support linter hints over it (you can check why both json/json5 work in "json.schemas.fileMatch" property of ".vscode/settings.json").
When you rename the file, it's more simple to understand that what you should do is just to add "extraResources" field to the root object of it, that is in "electon-builder.json":
{
...
  "extraResources": [
    {
      "from": "./yourOriginFolder",
      "to": "./yourDistribFolder"
    }
  ],
...
}
Hope it helps.
